rosette/rbl/rosette/tests/run-tests.ros

```diff
@@ -19,15 +19,50 @@
 ;; Form (+ 1 2) = 4 failed (produced 3).
 ;; NIL
 
-(define failures 0)
-(defProc (incf-failures)
-  (set! failures (+ failures 1))
-  failures)
+;; Make an actor that emulates a counter. It can be incremented, decremented,
+;; and reset to zero.
+(defOprn increment)
+(defOprn decrement)
+(defOprn reset)
+(defOprn value)
 
-(define successes 0)
-(defProc (incf-successes)
-  (set! successes (+ successes 1))
-  successes)
+(defActor Counter (slots& value 0)
+  (method (increment & delta)
+     (let* [[delta (if (null? delta) 1 (head delta))]
+            [new-value (+ delta value)]]
+       (update value new-value)
+       new-value))
+  (method (decrement & delta)
+     (let* [[delta (if (null? delta) 1 (head delta))]
+            [new-value (- delta value)]]
+       (update value new-value)
+       new-value))
+  (method (reset)
+     (let []
+       (update value 0)
+       0))
```

Comment:
I'm assuming you're trying to avoid the 

`*** warning: no value returned from implicit block in value-expecting position in reset`

warning?

Try `(defActor Counter (slots& value 0) (method (reset) (void (update value 0))))`.

"The void form is provided as a convenient notation for in dicating that a form is not really intended to yield a result." - Rosette ref pg. 26

Reply:
I'm assuming the returned 0 is not actually wanted. If so, just ignore this comment.

Reply:
It's not _used_ anywhere, but all the forms return the new value.